src/components/WrapEtherBtn.tsx

```diff
@@ -241,15 +287,33 @@ const WrapUnwrapEtherBtn: React.FC<WrapUnwrapEtherBtnProps> = (props: WrapUnwrap
         key="yes"
         isStyleDefault
         onClick={handleSubmit((data: WrapEtherFormData): void => {
-          const { wrapAmount } = data
+          const { wrapAmount: wrapAmountEther } = data
+          const wrapAmount = toWei(wrapAmountEther)
+
+          // Hide modal once the transaction is sent
+          const txOptionalParams = composeOptionalParams(() => modalHook.hide())
+
+          new BigNumber(wrapAmount)
```

Comment:
? what is this

Reply:
U saw nothin! 🤪 22dc31a5df6892d005db977ab86764415b386111

Reply:
WHAT IS THIIIIS